src/components/DaoList/DaoContainer.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 import Subscribe, { IObservableState } from "components/Shared/Subscribe"
-import { arc } from "arc"
+import { getArc } from 'arc'
```

Comment:
i thought we were changing everything to double quotes abut now you seem to be changing everything to single quotes

Reply:
My muscle memory types singe quotes. We need to enable the linter rule.  I've made an issue: https://github.com/daostack/alchemy/issues/295